src/jit/lsraarm64.cpp

```diff
@@ -981,105 +981,193 @@ int LinearScan::BuildSIMD(GenTreeSIMD* simdTree)
 //
 int LinearScan::BuildHWIntrinsic(GenTreeHWIntrinsic* intrinsicTree)
```

Comment:
There is, again, quite a bit of logic that is similar between the x86 and ARM paths in lsra and lowering (basically everything that isn't the individual intrinsic handling), so there might be a good opportunity to share code in the future.